plugins/src/test/kotlin/org/gradle/kotlin/dsl/plugins/AbstractPluginTest.kt

```diff
@@ -19,7 +19,7 @@ open class AbstractPluginTest : AbstractIntegrationTest() {
             pluginManagement {
                 repositories {
                     maven { setUrl("$testRepository") }
```

Comment:
:o: does it still have to be `setUrl(...)` here?

Reply:
Oh, definitely no! Good catch!